src/frontend/web_application/src/components/OpenPGPKey/index.jsx

```diff
@@ -0,0 +1,150 @@
+        <div className="m-openpgp-key__main">
+          <div className="m-openpgp-key__icon">{children}</div>
+          <div className="m-openpgp-key__fingerprint">{this.viewValues.fingerprint}</div>
+
+          <div className="m-openpgp-key__actions">
+            <Button
+              className={classnames({ 'm-openpgp-key__toggle-info--warning': this.viewValues.keyStatus !== 'valid' })}
+              onClick={this.toggleDetails}
+            >
+              <Icon type="info-circle" />
+              {' '}
+              <Icon type="caret-down" />
+              {' '}
+              <span className="show-for-sr">{__('openpgp.action.toggle-details')}</span>
+            </Button>
+            {editMode && (
+              <Button alert onClick={this.handleDeleteKey}>
+                <Icon type="remove" />
+                <span className="show-for-sr">{__('openpgp.action.remove-key')}</span>
+              </Button>
+            )}
+          </div>
+        </div>
+
+        {!this.state.showDetails && (
+          <div className="m-openpgp-key__summary">
+            <span>{this.viewValues.userId}</span>
+            {this.viewValues.createdAt && (
+              <DateTime format="ll">{this.viewValues.createdAt}</DateTime>
+            )}
+            {' '}
+            {this.viewValues.expirationTime && this.viewValues.expirationTime.length && (
+              <span>
+                / <DateTime format="ll">{this.viewValues.expirationTime}</DateTime>
+              </span>
+            )}
+            {' '}
+            {this.viewValues.keyStatus && openpgpStatuses[this.viewValues.keyStatus]}
+          </div>
+        )}
+
+        {this.state.showDetails && (
+          <div className="m-openpgp-key__details">
+            <DefList className="m-openpgp-key__detail">{[
+              { title: __('openpgp.details.identities'), descriptions: this.viewValues.userIds },
+              { title: __('openpgp.details.algorithm'), descriptions: [this.viewValues.algorithm] },
+              { title: __('openpgp.details.key-size'), descriptions: [this.viewValues.bitSize] },
+              { title: __('openpgp.details.status'), descriptions: [openpgpStatuses[this.viewValues.keyStatus]] },
+              { title: __('openpgp.details.creation'), descriptions: this.viewValues.createdAt ? [<DateTime format="ll">{this.viewValues.createdAt}</DateTime>] : [] },
+              { title: __('openpgp.details.expiration'), descriptions: this.viewValues.expirationTime ? [<DateTime format="ll">{this.viewValues.expirationTime}</DateTime>] : [] },
+            ]}</DefList>
+
+            <div className="m-openpgp-key__detail">
+              <textarea-field-group
+                label="'openpgp.public-key'|translate"
+                model={publicKeyArmored}
+              />
+            </div>
+            <div className="m-openpgp-key__detail">
+              <textarea-field-group
+                ng-if="!!$ctrl.privateKeyArmored"
```

Comment:
o_O, you right. And `textarea-field-group` is not migrated too. I don't understand what happened, I may missed that or a rebase failure. Thanks!